features/step_definitions/fields_steps.rb

```diff
@@ -1,4 +1,16 @@
+# frozen_string_literal: true
+
+# FIXME: this step should assert that input has required=true
 Then /^fields should be required:$/ do |table|
+  table.rows.each do |row|
+    field = row.first
+    assert find('label', text: field).has_css?('span.pf-c-form__label-required'),
+           "Field #{field} is not required"
+  end
+end
+
+# FIXME: this used in some liquid templates, it's wrong, remove it.
```

Comment:
Are these FIXMEs work in progress?

Reply:
This FIXME in particular not because it would affect many scenarios and maybe we don't even want to do it. But we shouldn't lose sigh of it.